packages/eventual-send/src/handled-promise.js

```diff
@@ -546,11 +544,6 @@ export const makeHandledPromise = () => {
         .catch(lose);
     });
 
-    // Harden the fulfillment and rejection, as well as a workaround for
-    // Node.js: silence "Unhandled Rejection" by default when using the static
-    // methods.
-    returnedP.then(harden, harden);
```

Comment:
Yes, we no longer need silencing because the catches are correct.

Reply:
Where do we harden the fulfillment value now?

Reply:
I no longer think that fulfilment needs to be hardened, since that feature is incompatible with avoiding silencing the unhandled rejections.

Reply:
Incompatible how? What about doing:

```js
returnedP.then(hardenVoid).catch(() => {});
```

Reply:
> Incompatible how?

See the challenge above.  Touching `returnedP` in any way breaks my requirements.

Reply:
What do you mean "touching it"?

@michaelfig , you and I talked since this thread and reached a decision about hardening. Do you remember what that was?